### PR TITLE
updated all links to img tag

### PR DESCRIPTION
**Pull request by sushi2k**
This PR is doing a QA of all pictures. Some pictures were not showing due to wrong filename/path, some pictures were still too big/small and all pictures are now using an image tag.

Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1156 .

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 0caf3f16be2f347ce38f0d6feec1c6ea2bbd687f: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 0d4efc5d2e053ddbe9543f32a9a77c4dcc8e7ef8: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -1164,11 +1166,11 @@ Many apps log informative (and potentially sensitive) messages to the console lo
 5. Reproduce the problem.
 6. Click on the **Open Console** button located in the upper right-hand area of the Devices window to view the console logs on a separate window.
 
-![Opening the Device Console in Xcode](Images/Chapters/0x06b/open_device_console.png)
+<img src="Images/Chapters/0x06b/open_device_console.png"  width="550">
```
Comment (cpholguera) [inline Document/0x06b-Basic-Security-Testing.md]:
```suggestion
<img src="Images/Chapters/0x06b/open_device_console.png"  width="550px">
```

File: Document/0x06b-Basic-Security-Testing.md [hunk 2]
```diff
@@ -1164,11 +1166,11 @@ Many apps log informative (and potentially sensitive) messages to the console lo
...
 
 To save the console output to a text file, go to the top right side of the Console window and click on the **Save** button.
 
-![Monitoring console logs through Xcode](Images/Chapters/0x06b/device_console.png)
+<img src="Images/Chapters/0x06b/device_console.png"  width="550">
```
Comment (cpholguera) [inline Document/0x06b-Basic-Security-Testing.md]:
```suggestion
<img src="Images/Chapters/0x06b/device_console.png"  width="550px">
```

File: Document/0x06b-Basic-Security-Testing.md [hunk 3]
```diff
@@ -1338,7 +1340,7 @@ Starting device <UDID> [SUCCEEDED] with interface rvi0
 ip.addr == 192.168.1.1 && http
 ```
 
-![Capture Filters in Wireshark](Images/Chapters/0x06b/wireshark_filters.png)
+<img src="Images/Chapters/0x06b/wireshark_filters.png" width="550">
```
Comment (cpholguera) [inline Document/0x06b-Basic-Security-Testing.md]:
```suggestion
<img src="Images/Chapters/0x06b/wireshark_filters.png" width="550px">
```

File: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md [hunk 4]
```diff
@@ -632,7 +632,7 @@ $ otool -Iv DamnVulnerableIOSApp | grep release
 
 IDB automates the processes of checking for stack canary and PIE support. Select the target binary in the IDB GUI and click the "Analyze Binary…" button.
 
-<img src="Images/Chapters/0x06i/idb.png" alt="IDB Analyze Binary" width="400">
+<img src="Images/Chapters/0x06i/idb.png" alt="IDB Analyze Binary" width="350">
```
Comment (cpholguera) [inline Document/0x06i-Testing-Code-Quality-and-Build-Settings.md]:
```suggestion
<img src="Images/Chapters/0x06i/idb.png" alt="IDB Analyze Binary" width="350px">
```

**Comment by OMTGreleaser** [comment]
Results for commit 70e3a94422869da5d5094eeefd29f5e056e98e17: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x05b-Basic-Security_Testing.md [hunk 5]
```diff
@@ -1747,7 +1747,8 @@ The interception proxy must listen to the port specified in the port forwarding
 
 Start the app and trigger a function that uses FCM. You should see HTTP messages in your interception proxy.
 
-![Intercepted Messages](Images/Chapters/0x05b/FCM_Intercept.png)
+<img src="Images/Chapters/0x05b/FCM_Intercept.png" width="500px"/>
+![Intercepted Messages]()
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document/0x05b-Basic-Security_Testing.md]: This is unfortunately not working. I created a ticket for this https://github.com/OWASP/owasp-mstg/issues/1577 and moved it to 1.3 release :-)

File: Document/0x05h-Testing-Platform-Interaction.md [hunk 6]
```diff
@@ -90,7 +90,11 @@ This allows a common capability-style model where user interaction drives ad-hoc
 
 #### Documentation for URI Permissions
 
-[grantUriPermission](https://developer.android.com/reference/android/content/Context.html#grantUriPermission(java.lang.String,%2520android.net.Uri,%2520int) "grantUriPermission"), [revokeUriPermission](https://developer.android.com/reference/android/content/Context#revokeUriPermission(android.net.Uri,%20int) "revokeUriPermission"), and [checkUriPermission](https://developer.android.com/reference/android/content/Context#checkUriPermission(android.net.Uri,%20int,%20int,%20int) "checkUriPermission").
+- [grantUriPermission](http://bit.ly/2Ke2AQU "grantUriPermission")
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document/0x05h-Testing-Platform-Interaction.md]: They are permanent, I created them under my bit.ly account.

File: Document/0x05d-Testing-Data-Storage.md [hunk 7]
```diff
@@ -864,7 +864,7 @@ While black-box testing the app, navigate to any screen that contains sensitive
 
 | `FLAG_SECURE` not set  | `FLAG_SECURE` set  |
 |---|---|
-| ![OMTG_DATAST_010_1_FLAG_SECURE](Images/Chapters/0x05d/1.png)   |  ![OMTG_DATAST_010_2_FLAG_SECURE](Images/Chapters/0x05d/2.png) |
+| <img src="Images/Chapters/0x05d/1.png" width="500px"/> | <img src="Images/Chapters/0x05d/2.png" width="500px"/> |
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document/0x05d-Testing-Data-Storage.md]: yes, it's scaled down in the PDF and looks good. just checked

**Comment by OMTGreleaser** [comment]
Results for commit 4ca60d2580dc51970ec755f4172fc1e9bf6636a2: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 8]
```diff
@@ -750,7 +750,9 @@ Different methods exist for installing an IPA package onto an iOS device, which
 
 ##### Cydia Impactor
 
-One tool that is available for Windows, macOS and Linux is [Cydia Impactor](http://www.cydiaimpactor.com/ "Cydia Impactor"). This tool was originally created to jailbreak iPhones, but has been rewritten to sign and install IPA packages to iOS devices via sideloading. The tool can even be used to install APK files to Android devices. A [step by step guide and troubleshooting steps can be found here](https://yalujailbreak.net/how-to-use-cydia-impactor/ "How to use Cydia Impactor").
+One tool that is available for Windows, macOS and Linux is [Cydia Impactor](http://www.cydiaimpactor.com/ "Cydia Impactor"). This tool was originally created to jailbreak iPhones, but has been rewritten to sign and install IPA packages to iOS devices via sideloading. A [step by step guide and troubleshooting steps are available on yalujailbreak.net](https://yalujailbreak.net/how-to-use-cydia-impactor/ "How to use Cydia Impactor").
+
+The tool can even be used to install APK files to Android devices.
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: agree

**Comment by OMTGreleaser** [comment]
Results for commit 9ca62219b0c330b8a1048fc90c027d4840f7aab7: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by sushi2k** [comment]
Fixed also JDB -> jdb :-)
Thanks @cpholguera. Think now it's ready to merge

**Comment by OMTGreleaser** [comment]
Results for commit 1e483b1b21b6cacd3af76108465c0f8a0084d02a: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit a5e45ec1aecfc9ed5f942136094a43340677b326: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**